app/controllers/api/notifications_controller.rb

```diff
@@ -13,5 +15,13 @@ def mark_as_seen_resource(type, id = nil, _data = nil)
         action_result(notification.update_attribute(:seen, true) || false)
       end
     end
+
+    # Would prefer not to override this method, may want to find another point
```

Comment:
Very interesting - I was pretty sure the controllers could define a way to do includes on resources, but I can't seem to find it.  I'm wondering if we could allow a helper method like how we do the `*_search_conditions`, but instead do something like `*_render_includes` ? 🤔

Reply:
Note that instead of overriding render_collection, the metrics and metrics_rollups controllers override index.  Not sure if that's better or not.

Reply:
yea. that is the only other way of overriding this.
I can look into this